easybuild/tools/filetools.py

```diff
@@ -700,8 +700,15 @@ def adjust_permissions(name, permissionBits, add=True, onlyfiles=False, onlydirs
         for root, dirs, files in os.walk(name):
             paths = []
             if not onlydirs:
-                paths += files
+                if not skip_symlinks:
+                    paths += files
```

Comment:
let's switch to `paths.extend(files)` here (matches better with the `.append` used below)